service/resource/deployment/deployment.go

```diff
@@ -44,7 +44,7 @@ func (r Resource) newMainDeployment(obj providerv1alpha1.AzureConfig) (Deploymen
 		n := node{
 			AdminUsername:   key.AdminUsername(obj),
 			AdminSSHKeyData: key.AdminSSHKeyData(obj),
-			OSImage:         newNodeOSImageCoreOS_1465_7_0(),
+			OSImage:         newNodeOSImageCoreOS_1576_5_0(),
```

Comment:
what is the purpose of version bundle if you're still hard coding CoreOS version like this?